openedx/features/ucsd_features/ecommerce/utils.py

```diff
@@ -0,0 +1,28 @@
+"""
+Util methods related to Open edX e-commerce service
+"""
+import logging
+import pygeoip
+
+from django.conf import settings
+
+from openedx.features.ucsd_features.ecommerce.EcommerceClient import EcommerceRestAPIClient
+
+
+logger = logging.getLogger(__name__)
+
+
+def is_user_eligible_for_discount(request, course_key):
+    if not settings.FEATURES['ENABLE_GEOGRAPHIC_DISCOUNTS']:
```

Comment:
```suggestion
    if not settings.FEATURES.get('ENABLE_GEOGRAPHIC_DISCOUNTS', None):
```

Reply:
not necessarily required as I am providing  a default value in config but let's change it anyways to be on safe side